frontend/src/sass/_variables.scss

```diff
@@ -0,0 +1,5 @@
S4wMTIgMi41NzE2OSwtMS4wMTIgMC4zNjA2MSwwIDEuNzEyNDYsMS4xMDA4IDMuMjUwNTIsMi42NDY4IDIuNzI2NzIsMi43NDEgMy4yMzczNywyLjk4MDkgNS4yMTU5NywyLjQ1MDcgeiBtIDEwLjY2NTQ5LC0yLjc3MjkgMCwtMTUuNzM2NyAzLjE5MTg2LDAuMDg5IDMuMTkxODYsMC4wODkgNi4zMDU4OCwxMC4yNjE3IDYuMzA1ODcsMTAuMjYxNyAwLC0xMC4zMzk1IDAsLTEwLjMzOTYgMi45NTgzMiwwIDIuOTU4MzEsMCAwLDE1LjczNjMgMCwxNS43MzYzIC0zLjMyNTA1LC0wLjA4OCAtMy4zMjUwNSwtMC4wODggLTUuOTM5MTQsLTkuNjIzNSAtNS45MzkxNCwtOS42MjM2IC0wLjA4MjMsOS43MDE1IC0wLjA4MjMsOS43MDEzIC0zLjEwOTU4LDAgLTMuMTA5NTgsMCAwLC0xNS43MzY2IHogbSAzOC42MTM3NiwyLjgxMzUgMCwtMTIuOTIzMiAtNC41MTUzMiwwIC00LjUxNTMyLDAgMCwtMi44MDI2IDAsLTIuODAyNiAxMi40NTYwNSwwIDEyLjQ1NjA1LDAgMCwyLjgwMjYgMCwyLjgwMjYgLTQuNjcxMDEsMCAtNC42NzEwMywwIDAsMTIuOTIzMiAwLDEyLjkyMzEgLTMuMjY5NzEsMCAtMy4yNjk3MSwwIDAsLTEyLjkyMzEgeiBtIDUyLjAwNCwtMi42NDcgMCwtMTUuNTcgMy4yNjk3MiwwIDMuMjY5NzIsMCAwLDEyLjkyMzEgMCwxMi45MjMyIDcuOTQwNzIsMCA3Ljk0MDczLDAgMCwyLjY0NjkgMCwyLjY0NjkgLTExLjIxMDQ0LDAgLTExLjIxMDQ1LDAgMCwtMTUuNTcwMSB6IG0gMzQuODc2OTQsMTQuOTk4OSBjIC0yLjgxOTk4LC0wLjkzOCAtNC4zMDQyNCwtMS44NTMxIC02LjM5MTg2LC0zLjk0MDcgLTIuMTMxNzUsLTIuMTMxOCAtMy4yMDk0MSwtMy45NTA4IC00LjA0NjQ1LC02LjgzMDEgLTAuNzM2NjYsLTIuNTM0IC0wLjUxNzgzLC03LjQzMTQgMC40NDI1MywtOS45MDMzIDEuNDk4NzEsLTMuODU3NSA1LjEzNDA3LC03LjQ5MjkgOC45OTE1OSwtOC45OTE1IDIuNDcxODcsLTAuOTYwNCA3LjM2OTI2LC0xLjE3OTIgOS45MDMzMywtMC40NDI2IDIuODc5OTgsMC44MzczIDQuNjk4MzgsMS45MTQ4IDYuODMyMzksNC4wNDg5IDMuNDE4NDMsMy40MTg0IDQuNjQzNDUsNi43MDE0IDQuNDYwNzYsMTEuOTU0OCAtMC4xMDYzLDMuMDU3MSAtMC4yMDM2LDMuNDk1NiAtMS4yNzU2LDUuNzQ4NiAtMS41ODAzNCwzLjMyMTQgLTQuMjc3NjMsNi4wMTk3IC03LjYwMjcyLDcuNjA1NCAtMi4zMjY4NiwxLjEwOTggLTIuNjAzNzMsMS4xNjY4IC02LjAyMDE0LDEuMjQxNiAtMi43OTY4MywwLjA2MSAtMy45NTYyMSwtMC4wNDYgLTUuMjkzODMsLTAuNDkxMSB6IG0gMTAuNTg3NjUsLTUuNTAxMSBjIDAuNDI4MTcsLTAuMjQ0OCAxLjA5Mzc5LC0wLjQ1IDEuNDc5MTUsLTAuNDU2MSAwLjY5MDI1LC0wLjAxMSAwLjcwMDY2LC0wLjA3MSAwLjcwMDY2LC00LjA3NTMgbCAwLC00LjA2NDYgLTAuOTc0NDIsLTAuMzM5NiBjIC0wLjczMzI5LC0wLjI1NTYgLTEuNDI2ODQsLTEuMDM5MyAtMi44MDI2MSwtMy4xNjY2IC0xLjczNzY3LC0yLjY4NzEgLTEuODI4MiwtMi45MjQzIC0xLjgyODIsLTQuNzkwMSAwLC0xLjgzMTUgLTAuMDU0NywtMS45ODM3IC0wLjgxNjM5LC0yLjI3MzMgLTEuMDg3NjIsLTAuNDEzNSAtMS40OTQxNCwtMC4wMzcgLTIuMzA2MjksMi4xMzMyIC0wLjU3NzQ4LDEuNTQzNCAtMC42MTU3MSwxLjk3NjEgLTAuMzIzMTUsMy42NTYgMC4xODI1LDEuMDQ4MSAwLjMzMTgyLDEuOTk5MiAwLjMzMTgyLDIuMTEzNiAwLDAuMTE0NCAtMS41MTM0MiwwLjIwOCAtMy4zNjMxMywwLjIwOCAtMi4yMDA1OCwwIC0zLjQ5MjMxLDAuMTI5MiAtMy43MzY4MiwwLjM3MzcgLTAuNDYxODUsMC40NjE4IC0wLjQ5MTY2LDIuMDA3OSAtMC4wNDQsMi4yODQ2IDAuMTkwNTksMC4xMTc3IDAuMjQ0NjUsMC41OTA2IDAuMTI4MTUsMS4xMjEgLTAuMTI1OTcsMC41NzM1IC0wLjA0ODMsMS4xMDE4IDAuMjA3MTQsMS40MDk3IDAuMjI0NzUsMC4yNzA3IDAuNDQwMTQsMS4wMDA3IDAuNDc4NjQsMS42MjIgMC4yMTUxNywzLjQ3MTUgMC4zNzkwNywzLjk2NjYgMS40NTczNiw0LjQwMTYgMC4zNjc4NCwwLjE0ODQgMi45MTA4OCwwLjI3MzggNS42NTEyMSwwLjI3ODYgMy43NjExMSwwLjAxIDUuMTczMjUsLTAuMTAwNCA1Ljc2MDkyLC0wLjQzNjQgeiIKICAgICAgIGlkPSJwYXRoMzAwMCIKICAgICAgIHN0eWxlPSJmaWxsOiNlNjdlMjI7ZmlsbC1vcGFjaXR5OjEiIC8+CiAgICA8cGF0aAogICAgICAgZD0iIgogICAgICAgaWQ9InBhdGgyOTk2IgogICAgICAgc3R5bGU9ImZpbGw6IzAwMDAwMCIgLz4KICA8L2c+Cjwvc3ZnPgo='
```

Comment:
can we move this so-called variable to separate file? it's so long, that it just shadow any other variable, that will be stored here.

Reply:
@magul Yes, I can do that, that would be good idea. 

However I am still thinking whether it is a good solution to keep the image in css or not. One one hand, css gets downloaded and then parsed, so we have instant access to the base64 image on parsing stage and don't need to wait for a server to fetch the image as it would be with URL. 

On the other hand, the image would need to be downloaded in the 'normal' format for our logo in Angular app, which gives a data overhead. Base64 of this image is 11kB, normal image is 8,5kB. 

If we use a URL to server image in css instead, then the image is downloaded only once, since images get cached in a browser and can be loaded immediately in Angular app.

What is your opinion on that?